frontend-ssr/package.json

```diff
@@ -40,6 +40,7 @@
     "highlight.js": "^10.4.1",
```

Comment:
You should still have the `yarn.lock` updated - meaning you should ru `yarn` inside the project and commit those changes as well.